### PR TITLE
Init GeoStyler workshop abstract

### PR DESCRIPTION
**Pull request by KaiVolland**

### CODE REVIEW

File: abstracts/workshops/geostyler.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+In diesem Workshop lernen Sie die ersten Schritte mit dem GeoStyler und bekommen einen
+Einblick in die Möglichkeiten, die der GeoStyler mit seinen Komponenten bietet.
+
+Der GeoStyler bietet ein ReactJS basiertes User Interface zur Erstellung von
+geographischen Stilinformationen auf Layer-Ebene. Dies umfasst unter anderem
+SLD, OpenLayers- und QGIS-Stile.
+
+Die Teilnehmer des Workshops lernen anhand einfacher Beispiele wie der GeoStyler
+und einzelne Komponenten in eine existierende Webanwendung integriert werden können oder als
```
Comment (jansule) [inline abstracts/workshops/geostyler.md]:
```suggestion
und einzelne Komponenten in existierende Webanwendungen integriert werden können oder als
```

**Review by jansule — APPROVED** [review]
We should be more explicit what GeoStyler is and maybe also add more details about the target group and what we want to achieve. Maybe something along the lines of 

> GeoStyler bietet die Möglichkeit vorgefertigte Styling Komponenten nahtlos in eigene Webanwendungen zu integrieren um Nutzern das Styling von Geodaten über eine graphische Schnittstelle zu ermöglichen. Dies trägt dazu bei, eigene WebGis-Anwendungen auf einfache Art zu erweitern...

> Dieser Workshop richtet sich an...

**Comment by chrismayer** [comment]
I am with @jansule that it should become more clear what the goal of the WS is. One could think that a GeoStyler workshop is about creating and transforming styles with the UI. As I understood the WS is targeting more the programming with the GeoStyler lib (which is totally fine of course).

File: abstracts/workshops/geostyler.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Leitung: 
+
+## Einreichen von: 
+
+## Abstract
+
+In diesem Workshop lernen Sie die ersten Schritte mit dem GeoStyler und bekommen einen
+Einblick in die Möglichkeiten, die der GeoStyler mit seinen Komponenten bietet.
+
+Der GeoStyler bietet ein ReactJS basiertes User Interface zur Erstellung von
```
Comment (jansule) [inline abstracts/workshops/geostyler.md]:
```suggestion
Der GeoStyler bietet ein anpassbares ReactJS basiertes User Interface zur Erstellung von
```

File: abstracts/workshops/geostyler.md (hunk 1 above)
Comment (jansule) [inline abstracts/workshops/geostyler.md]:
```suggestion
und einzelne Komponenten nahtlos in eine existierende Webanwendung integriert werden können oder als
```

File: abstracts/workshops/geostyler.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+geographischen Stilinformationen auf Layer-Ebene. Dies umfasst unter anderem
+SLD, OpenLayers- und QGIS-Stile.
+
+Die Teilnehmer des Workshops lernen anhand einfacher Beispiele wie der GeoStyler
+und einzelne Komponenten in eine existierende Webanwendung integriert werden können oder als
+Standalone Applikation verwendet werden kann.
```
Comment (jansule) [inline abstracts/workshops/geostyler.md]:
```suggestion
Standalone Applikation verwendet werden kann. Die vorgefertigten Styling Komponenten ermöglichen so, auf einfache Art eigene WebGIS-Anwendungen zu erweitern und Nutzern das Styling von Geodaten über eine graphische Schnittstelle zu erlauben.
```

File: abstracts/workshops/geostyler.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Vorkenntnisse in JavaScript oder ReactJS sind hilfreich, aber nicht zwingend erforderlich.
```
Comment (jansule) [inline abstracts/workshops/geostyler.md:21]:
```suggestion
Dieser Workshop richtet sich an (zukünftige) Entwickler von WebGIS-Anwendungen, die die Möglichkeit bieten sollen, Nutzern das Styling von Geodaten zu erlauben. 
Vorkenntnisse in JavaScript oder ReactJS sind hilfreich, aber nicht zwingend erforderlich.
```

**Review by jansule — CHANGES_REQUESTED** [review]

File: abstracts/workshops/geostyler.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+In diesem Workshop lernen Sie die ersten Schritte mit dem GeoStyler und bekommen einen
+Einblick in die Möglichkeiten, die der GeoStyler mit seinen Komponenten bietet.
+
+Der GeoStyler bietet ein anpassbares ReactJS basiertes User Interface zur Erstellung von
+geographischen Stilinformationen auf Layer-Ebene. Dies umfasst unter anderem
+SLD, OpenLayers- und QGIS-Stile.
+
+Die Teilnehmer des Workshops lernen anhand einfacher Beispiele wie der GeoStyler
+und einzelne Komponenten nahtlos in eine existierende Webanwendung integriert werden können oder als
```
Comment (jansule) [inline abstracts/workshops/geostyler.md]:
```suggestion
und einzelne Komponenten nahtlos in existierende Webanwendungen integriert werden können oder als
```

**Review by jansule — APPROVED** [review]

**Comment by chrismayer** [comment]
Thanks @KaiVolland !

**Pull request merged**